languages.toml

```diff
@@ -381,6 +381,19 @@ indent = { tab-width = 2, unit = "  " }
 name = "css"
 source = { git = "https://github.com/tree-sitter/tree-sitter-css", rev = "94e10230939e702b4fa3fa2cb5c3bc7173b95d07" }
 
+[[grammar]]
+name = "scss"
+source = { git = "https://github.com/serenadeai/tree-sitter-scss.git", rev = "c478c6868648eff49eb04a4df90d703dc45b312a" }
```

Comment:
This is a bit of a nitpick but could you move this below the `[[language]]` declaration for scss? It's not functionally different - just for consistency's sake

Reply:
I embarassingly fiddled with that a few times and still managed to land on the wrong order